magento/magento2
# Fix bulk action log clean error on admin dashboard

### Description (*)


This pull request fixes the error that happens when the admin user clicks the "Dismiss All Completed Tasks", preventing the tasks notice from being cleaned.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#36255

### Manual testing scenarios (*)

Steps to reproduce:

1. Log as an admin user.
2. Perform a Bulk update product attributes: 
  a. Catalog -> Products -> Select a N number of prodcuts and select the "Update Attributes" action. Update any number of attributes and submit the action.
3. Make the operation completed (eg: run `bin/magento queue:consumer:start product_action_attribute.update`)
4. Go back to the dashboard and click on the "Dismiss All Completed Tasks" link after expanding the admin notification section.
![image](https://user-images.githubusercontent.com/15946418/197263662-1cf87cd1-cf1c-4967-b71a-2aea7f4d5b37.png)
5. The messages should be cleared successfully with no error messages or popups.

### Contribution checklist (*)
